examples/step-7/step-7.cc

```diff
@@ -64,11 +64,7 @@
 // the same file as the FEValues class:
 #include <deal.II/fe/fe_values.h>
 
-// We need one more include from standard C++, which is necessary when we try
-// to find out the actual type behind a pointer to a base class. We will
-// explain this in slightly more detail below. The other two include files are
-// obvious then:
-#include <typeinfo>
+// This is C++:
```

Comment:
What about just removing the comment entirely? I tried to understand what you meant by "this is C++" now for 3 minutes.